sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_visits_daily_v2/query.sql

```diff
@@ -0,0 +1,284 @@
+          'topic_selection_topics_saved',
+          -- section
+          'sections_block_section',
+          'sections_follow_section',
+          'sections_unblock_section',
+          'sections_unfollow_section',
+          'inline_selection_click'
+        )
+      )
+    )
+)
+SELECT
+  submission_date,
+  client_id,
+  mozfun.map.get_key(event_details, 'newtab_visit_id') AS newtab_visit_id,
+  -- ANY_VALUE: because we do not expect those fields to differ within a visit
+  ANY_VALUE(app_version) AS app_version,
+  ANY_VALUE(os) AS os,
+  ANY_VALUE(channel) AS channel,
+  ANY_VALUE(locale) AS locale,
+  ANY_VALUE(country) AS country,
+  ANY_VALUE(homepage_category) AS homepage_category,
+  ANY_VALUE(newtab_category) AS newtab_category,
+  ANY_VALUE(organic_content_enabled) AS organic_content_enabled,
+  ANY_VALUE(sponsored_content_enabled) AS sponsored_content_enabled,
+  ANY_VALUE(sponsored_topsites_enabled) AS sponsored_topsites_enabled,
+  ANY_VALUE(organic_topsites_enabled) AS organic_topsites_enabled,
+  ANY_VALUE(newtab_search_enabled) AS newtab_search_enabled,
+  LOGICAL_OR(is_default_ui) AS is_default_ui,
+  LOGICAL_OR(event_category = 'newtab' AND event_name = 'opened') AS is_newtab_opened,
+  LOGICAL_OR(event_category = 'newtab.search' AND event_name = 'issued')
+  AND LOGICAL_OR(is_default_ui) AS is_search_issued,
+  LOGICAL_OR(event_category = 'newtab.search.ad' AND event_name = 'click')
+  AND LOGICAL_OR(is_default_ui) AS is_search_ad_click,
+  LOGICAL_OR(
+    event_category = 'pocket'
+    AND event_name IN ('click', 'dismiss', 'thumb_voting_interaction', 'thumb_voting_interaction')
+  )
+  AND LOGICAL_OR(is_default_ui) AS is_content_interaction,
+  LOGICAL_OR(event_category = 'pocket' AND event_name IN ('click'))
+  AND LOGICAL_OR(is_default_ui) AS is_content_click,
+  LOGICAL_OR(event_category = 'pocket' AND event_name IN ('impression'))
+  AND LOGICAL_OR(is_default_ui) AS is_content_impression,
+  LOGICAL_OR(
+    event_category = 'pocket'
+    AND event_name IN ('click', 'dismiss')
+    AND SAFE_CAST(mozfun.map.get_key(event_details, 'is_sponsored') AS BOOLEAN)
+  )
+  AND LOGICAL_OR(is_default_ui) AS is_sponsored_content_interaction,
+  LOGICAL_OR(
+    event_category = 'pocket'
+    AND event_name IN ('click')
+    AND SAFE_CAST(mozfun.map.get_key(event_details, 'is_sponsored') AS BOOLEAN)
+  )
+  AND LOGICAL_OR(is_default_ui) AS is_sponsored_content_click,
+  LOGICAL_OR(
+    event_category = 'pocket'
+    AND event_name IN ('impression')
+    AND SAFE_CAST(mozfun.map.get_key(event_details, 'is_sponsored') AS BOOLEAN)
+  )
+  AND LOGICAL_OR(is_default_ui) AS is_sponsored_content_impression,
+  IF(
+    LOGICAL_OR(is_default_ui),
+    COUNTIF(event_category = 'pocket' AND event_name IN ('click')),
+    0
+  ) AS any_content_click_count,
+  IF(
+    LOGICAL_OR(is_default_ui),
+    COUNTIF(event_category = 'pocket' AND event_name IN ('impression')),
+    0
+  ) AS any_content_impression_count,
```

Comment:
Nit: For this table I think `total_*` makes more sense than `any_*` here

Reply:
I believe those were the field names as per the [spec](https://docs.google.com/document/d/1UJpGhy6ZPW96eblZOd0WyP7N5VBVgN0aXi5T8GzqWU8/edit?tab=t.0)

Reply:
I feel bad for not catching it the spec but it might be worth considering a name change at this point.  I'd like to hear what @florezita and Ilana think

Reply:
I feel the same thoughts mentioned [here](https://github.com/mozilla/bigquery-etl/pull/7362/files#r2093356671)

Reply:
I am going to mark this as resolved. Unless @florezita tells me, changing the prefix for those fields is critical.